src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/FootnoteServiceTests.cs

```diff
@@ -538,11 +544,490 @@ void AssertFootnoteDetailsCopiedCorrectly(Footnote originalFootnote, Footnote ne
+            };
+
+            var footnote = new Footnote
+            {
+                Content = "Original footnote",
+                Releases = new List<ReleaseFootnote>
+                {
+                    new ReleaseFootnote
+                    {
+                        ReleaseId = release.Id,
+                    }
+                },
+                Subjects = new List<SubjectFootnote>(),
+                Filters = new List<FilterFootnote>(),
+                FilterGroups = new List<FilterGroupFootnote>(),
+                FilterItems = new List<FilterItemFootnote>(),
+                Indicators = new List<IndicatorFootnote>(),
+            };
+
+            var releaseSubject = new ReleaseSubject
+            {
+                Release = release,
+                Subject = subject,
+            };
+
+            var contextId = Guid.NewGuid().ToString();
+            await using (var statisticsDbContext = InMemoryStatisticsDbContext(contextId))
+            {
+                await statisticsDbContext.AddRangeAsync(
+                    release,
+                    footnote,
+                    filter,
+                    indicatorGroup,
+                    releaseSubject);
+                await statisticsDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = InMemoryApplicationDbContext(contextId))
+            {
+                await contentDbContext.AddAsync(new Content.Model.Release
+                {
+                    Id = release.Id,
+                });
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            var dataBlockService = new Mock<IDataBlockService>(Strict);
+
+            dataBlockService.Setup(mock => mock.InvalidateCachedDataBlocks(release.Id))
+                .Returns(Task.CompletedTask);
+
+            await using (var statisticsDbContext = InMemoryStatisticsDbContext(contextId))
+            await using (var contentDbContext = InMemoryApplicationDbContext(contextId))
+            {
+                var service = SetupFootnoteService(
+                    statisticsDbContext,
+                    contentDbContext,
+                    dataBlockService: dataBlockService.Object);
+
+                var result = await service.UpdateFootnote(
+                    release.Id,
+                    footnote.Id,
+                    "Updated footnote",
+                    new List<Guid>{ filter.Id },
+                    new List<Guid>{ filterGroup.Id },
+                    new List<Guid>{ filterItem.Id },
+                    new List<Guid>{ indicator.Id },
+                    new List<Guid>{ subject.Id });
+
+                result.AssertRight();
+            }
+
+            MockUtils.VerifyAllMocks(dataBlockService);
+
+            await using (var statisticsDbContext = InMemoryStatisticsDbContext(contextId))
+            {
+                var dbFootnote = Assert.Single(statisticsDbContext.Footnote.ToList());
```

Comment:
Not a big deal, but 'saved' could be a better prefix than 'db' e.g. `savedFootnote`. Could do this for the others as well.